ginga/rv/plugins/LineProfile.py

```diff
@@ -275,45 +303,55 @@ def _slice(self, naxes, mk):
 
         # For axis > 3
         for i in range(2, naxes):
-            slice_obj[i] = int(round(self.image.revnaxis[i-2] + 1))
+            if i != i_sel:
+                slice_obj[i] = 0  # int(round(self.image.revnaxis[i-2] + 1))
```

Comment:
I really don't know what `int(round(self.image.revnaxis[i-2] + 1))` is for but it breaks when the cube has more than 3 dimensions. Setting to 0 fixes it but does it change the plugin behavior?